plugins/woocommerce-admin/client/customize-store/index.tsx

```diff
@@ -511,6 +530,49 @@ export const customizeStoreStateMachineDefinition = createMachine( {
 	},
 } );
 
+declare global {
+	interface Window {
+		__wcCustomizeStore: {
+			isFontLibraryAvailable: boolean | null;
+		};
+	}
+}
+
+// HACK: This is a temporary solution to pass flags computed into the iframe instance state machines.
+// This is needed because the iframe loads the entire Customize Store app. This means that the iframe instance will have different state machines than the parent window.
+const setFlagsForIframeInstance = async (
```

Comment:
Here we are not using the return value, should we just return void? @gigitux

Reply:
Thanks for the feedback! Fixed with 604475c39eb8778847e9fae7e8c35272271d80d7.